src/test/java/uk/gov/hmcts/reform/jobscheduler/controllers/jobscontroller/CreateTest.java

```diff
@@ -0,0 +1,55 @@
+package uk.gov.hmcts.reform.jobscheduler.controllers.jobscontroller;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.ResultActions;
+import uk.gov.hmcts.reform.jobscheduler.SampleData;
+import uk.gov.hmcts.reform.jobscheduler.model.Job;
+import uk.gov.hmcts.reform.jobscheduler.services.JobsService;
+
+import static org.hamcrest.core.StringEndsWith.endsWith;
+import static org.mockito.BDDMockito.given;
+import static org.mockito.Matchers.any;
+import static org.springframework.http.HttpHeaders.LOCATION;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.header;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+
+@RunWith(SpringRunner.class)
+@WebMvcTest
+public class CreateTest {
```

Comment:
I'd add another test checking if the job was actually created.

Reply:
There are not reads yet. I'd rather not test implementation here (verify given method on mock was called)

Reply:
Side effects can and, in my opinion, should be verified in unit tests, but I understand there's no one correct approach.